core/src/test/java/org/pentaho/platform/scheduler2/action/ActionRunnerTest.java

```diff
@@ -38,6 +38,7 @@
 import org.pentaho.platform.engine.core.system.PentahoSystem;
```

Comment:
Could do better on testing for ActionRunner, will look into it for future efforts.